test/ActivityRepository-test.js

```diff
@@ -57,25 +57,14 @@ describe('UserRepository', function() {
     expect(activityRepo).to.be.an.instanceOf(ActivityRepository);
   });
 
-  it('should organize user activity by date', function() {
-    expect(activityRepo.allUserLog).to.be.an('object');
-    expect(activityRepo.allUserLog).to.have.all.keys(['2019/06/15','2019/06/16']);
+  it('should store activity logs for all users', function() {
```

Comment:
You can ignore this, which was done when I thought logs would be intantiated in their corresponding repos.

This commit is reversed in the following commit.

Reply:
got it! Thanks for the clarification.